domains/Discussions/Database/Migrations/2020_10_12_104113_create_questions_table.php

```diff
@@ -19,7 +19,6 @@ public function up(): void
             $table->softDeletesTz();
             $table->timestampTz('resolved_at')->nullable();
 
-            $table->index('author_id');
```

Comment:
Why is this being removed?

Reply:
it should not. (A rebase removed it but I added it back.)
Not sure why it shows as removed since it is there in my branch....
I'll check